crates/solidity/testing/snapshots/cst_output/SourceUnit/everything/generated/0.4.11-failure.yml

```diff
@@ -60,5 +60,5 @@ Tree:
               - PathImport (Rule): # 30..40 ' "foo.sol"'
                   - AsciiStringLiteral (Token): '"foo.sol"' # 31..40
               - Semicolon (Token): ";" # 40..41
-      - EndOfFileTrivia (Rule): "\n" # 42..43
+      - EndOfFileTrivia (Rule): # 42..43 "\n"
```

Comment:
Previously, the `TestNodeBuilder` would not construct the whitespace children tokens and the printer logic would recognize this as an childless node, whereas the new printer does not, as we only skipped the leading/trailing trivia nodes before.

This should probably be fixed to also not include the `EndOfFileTrivia` much like the Leading/Trailing variants, however this is not as important as we will change the trivia model soon (DSL v2).

Reply:
See 47663c9313726734550b4c87bbef2208f1d5bf0b for the fix